sitemap.xml

```diff
@@ -7,23 +7,19 @@ search: exclude
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
   {% for post in site.posts %}
   {% unless post.search == "exclude" %}
-  {% if site.url and post.url %}
   <url>
-    <loc>{{site.url}}{{post.url}}</loc>
```

Comment:
Why those checks were removed?

Reply:
they are not needed, they were introduced in my previous PR

Reply:
Which one?

Reply:
both

Reply:
https://github.com/spryker/spryker-docs/pull/2809